example/serializers.py

```diff
@@ -260,6 +260,12 @@ class Meta:
     def get_first_entry(self, obj):
         return obj.entries.first()
 
+class AuthorListSerializer(AuthorSerializer):
```

Comment:
The reason the openapi components/schemas name changed is because your subclassed view is named `AuthorListSerializer`, and the class it extends is `AuthorSerializer`. DRF generates a default schema name by stripping `Serializer` off the class name.

Reply:
Ah, thanks for the explanation. Does that imply I simply should change the snapshots accordingly?

Reply:
Fixed...

Reply:
Yes, change the snapshots, or declare your class as AuthorSerializer...

Reply:
done